filetypes.md

```diff
@@ -0,0 +1,56 @@
+| Mime type                                                                 | Hoover Filetype | Analyzers      |
+| ---                                                                       | ---             | ---            |
+| application/x-7z-compressed                                               | archive         | archives       |
```

Comment:
Please add another column called `About` where you display a pretty name / explanation for the mime type. You can pull these from the mozilla list or wikipedia or anywhere else online


- https://www.freeformatter.com/mime-types-list.html
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types